backend/api/payments.py

```diff
@@ -0,0 +1,79 @@
+import math
+
+from backend.common import models
+from backend.common import config_info
+
+db = config_info.get_db()
+
+
+def create_payments(bill: models.Bill):
+    """
+    # Create payments
+    This function creates payments for a bill.
+    """
+    total_amounts_to_pay = {}
+    if bill.bill_type == models.BillType.SPLIT_BY_MEMBERS:
+        for payer in bill.payer_ids:
+            if payer.user_id not in total_amounts_to_pay:
+                total_amounts_to_pay[payer.user_id] = 0
+            total_amounts_to_pay[payer.user_id] += payer.amount
+        print(f"{total_amounts_to_pay = }")
+    elif bill.bill_type == models.BillType.SPLIT_BY_PRODUCTS:
+        for product in bill.products:
+            for payer in product.assigned_payers:
+                if payer.user_id not in total_amounts_to_pay:
+                    total_amounts_to_pay[payer.user_id] = 0
+                total_amounts_to_pay[payer.user_id] += payer.amount
+        print(f"{total_amounts_to_pay = }")
+
+    total_amount_to_receive = 0
```

Comment:
Try to use different names for the dict and this variable, they’re too similar and the code is already complicated enough to follow